setup.py

```diff
@@ -9,7 +9,7 @@
 # NOTE: This file must remain Python 2 compatible for the foreseeable future,
 # to ensure that we error out properly for people with outdated setuptools
 # and/or pip.
-min_version = (3, 7)
+min_version = (3, 9)
```

Comment:
It would be helpful to document somewhere why 3.8 no longer works.

Reply:
Added https://github.com/bluesky/bluesky-adaptive/issues/38